core/src/test/java/org/mule/el/datetime/DateTimeTimeTestCase.java

```diff
@@ -80,118 +84,112 @@ public void format()
     {
         SimpleDateFormat df = new SimpleDateFormat("hh:mm:ss");
         df.setTimeZone(TimeZone.getTimeZone("UTC"));
-        Assert.assertEquals(df.format(now.toDate()), now.format("hh:mm:ss"));
+        assertThat(now.format("hh:mm:ss"), is(df.format(calendarNow.getTime())));
     }
 
     @Test
     public void timeZone()
     {
-        Assert.assertEquals(TimeZone.getTimeZone("UTC").getDisplayName(), now.getTimeZone());
+        assertThat(now.getTimeZone(), is(TimeZone.getTimeZone("UTC").getDisplayName()));
     }
 
     @Test
     public void plusSeconds()
     {
-        Assert.assertEquals((Calendar.getInstance().get(Calendar.SECOND) + 1) % 60, now.plusSeconds(1)
-            .getSeconds());
+        assertThat(now.plusSeconds(1).getSeconds(), is((Calendar.getInstance().get(Calendar.SECOND) + 1) % 60));
     }
 
     @Test
     public void plusSecondsRollover()
     {
-        Assert.assertEquals(1, now.plusHours(172800).toCalendar().get(Calendar.DAY_OF_YEAR));
+        assertThat(now.getTime().plusHours(172800).toCalendar().get(Calendar.DAY_OF_YEAR), is(1));
```

Comment:
I'm not getting this test... the name refers to Seconds, but it adds Hours, and then compare DayOfYear?